dora/services/views.py

```diff
@@ -262,6 +266,13 @@ def perform_update(self, serializer):
         ):
             self._send_service_published_notification(service)
 
+        # Increment filling_duration if service become pusblished or is still a draft
```

Comment:
```suggestion
        # Increment filling_duration if service become published or is still a draft
```
Mais encore mieux en français ;)

Reply:
👍